### PR TITLE
Sealions working CRUD

### PR DESCRIPTION
**Pull request by foxdoubt**
@linkblaine @queerviolet

### CODE REVIEW

File: app/controllers/sealions.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+require 'sinatra'
+
+get '/sealions/new' do
+  erb :new
+end
+
+get '/sealions/:id/edit' do
+  lion_to_update = Sealion.find_by_id(params[:id])
```
Comment (cdimartino) [inline app/controllers/sealions.rb:8]: Sealion.find(params[:id]) would work too.  AR::find searches by id.

File: app/controllers/sealions.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+get '/sealions/:id' do
+  lion = Sealion.find_by_id(params[:id])
+  if lion
+    erb :show, locals: {cur_lion: lion}
+  else
+    [500, "Something went wrong. Sorry!"]
+  end
+end
+
+post '/sealions' do
```
Comment (cdimartino) [inline app/controllers/sealions.rb:27]: Looks good :+1:

File: app/controllers/sealions.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    redirect "sealions/#{new_lion.id}"
+  else
+    [500, "Something went wrong. Sorry!"]
+  end
+end
+
+put '/sealions/:id' do
+  cur_lion = Sealion.find_by(id: params[:id])
+
+  if cur_lion
```
Comment (cdimartino) [inline app/controllers/sealions.rb:39]: We can do this more concisely:

``` ruby
cur_lion.update(name: params[:name], age: params[:age], quirk: params[:quirk])
```

File: spec/controllers/sealions_spec.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  it 'Should process the delete/sea_delete route' do
+    delete '/sea_delete'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Welcome to the Sealions page!')
+  end
+end
```
Comment (cdimartino) [inline spec/controllers/sealions_spec.rb:34]: These tests look great :+1:

**Comment by cdimartino** [comment]
Looking good Dan!

File: spec/controllers/sealions_spec.rb (hunk 4 above)
Comment (foxdoubt) [inline spec/controllers/sealions_spec.rb:34]: Thanks @cdimartino for reading over my code!

**Pull request closed without merging**